### PR TITLE
Kind

### PR DESCRIPTION
**Pull request by marek-jakubowski**
Goal of this PR Is to provide additional way of deploying K8S-Follower.

Right now follower are created as additional compose container.
This PR introduces way of deploying Follower in Kubernetes cluster using Kind.
Details are defined in README.md file under artifacts/kind directory.

### CODE REVIEW

File: artifacts/kind/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+FROM registry.tld/cyberark/conjur-kubernetes-follower-operator-manifests:$K8S_FOLLOWER_TAG AS operator-manifests
+
+FROM golang:alpine
+
+ARG K8S_FOLLOWER_TAG
+
+RUN apk add docker-cli kubectl curl jq && \
+    go install sigs.k8s.io/kind@v0.20.0
+
+WORKDIR /
```
Comment (hdabrowski) [inline artifacts/kind/Dockerfile:12]: is it necessary, isn't it default value?

Comment (marek-jakubowski) [reply artifacts/kind/Dockerfile]: Thanks, removed.

Comment (marek-jakubowski) [reply artifacts/kind/Dockerfile:12]: Actually decided to leave it, because golang image points first to `/go`

**Review by tarnowsc — APPROVED** [review]
LGTM!

**Pull request merged**